extensions/amp-social-share/1.0/social-share-config.js

```diff
@@ -36,99 +40,104 @@ let SocialShareConfigDef;
  * contains the following properties:
  *   {string} shareEndpoint - The base API endpoint for sharing to the
  *     specified social media type.
- *   {JsonObject} defaultParams - Parameters to be appended to the end of the
+ *   {Object} defaultParams - Parameters to be appended to the end of the
  *     shareEndpoint as query parameters.  The values in this object are used
  *     as binding keys which are resolved by the AMP framework.
- * @type {!Object<string, !SocialShareConfigDef>}
+ *   {string} defaultColor - Default color code for this social media type.
+ *   {string} defaultBackgroundColor - Default background color code for this
+ *     social media type.
+ *   {?Array<string>} bindings - Used for email, allows passing in an
+ *     attribute that be used in endpoint, but not as a search param
+ * @type {Object<?string, SocialShareConfigDef>}
  */
 const BUILTINS = {
   'twitter': {
-    'shareEndpoint': 'https://twitter.com/intent/tweet',
-    'defaultParams': dict({
-      'text': 'TITLE',
-      'url': 'CANONICAL_URL',
-    }),
-    'defaultColor': 'ffffff',
-    'defaultBackgroundColor': '1da1f2',
+    shareEndpoint: 'https://twitter.com/intent/tweet',
+    defaultParams: {
+      text: 'TITLE',
```

Comment:
Because the `defaultParams` is defined as `!Object`, we should also use `dict()` and quotes. You can think of it this way: you don't define `text` and `url` fields anywhere - they are simply names. Some other name could be "this is the text" and it's still a reasonably-allowable name, even if unlikely.

Reply:
Ah thanks, so to confirm anytime the keys of the object would be the value of a variable (not the name of the variable), we should use `dict()` and quotes for that object? 

I've updated in the latest revision.